conjure-java-core/src/main/java/com/palantir/conjure/java/types/BeanBuilderGenerator.java

```diff
@@ -394,6 +409,7 @@ private boolean isByteBuffer(Type type) {
     private MethodSpec createOptionalSetter(EnrichedField enriched) {
         OptionalType type = enriched.conjureDef().getType().accept(TypeVisitor.OPTIONAL);
         return BeanBuilderAuxiliarySettersUtils.createOptionalSetterBuilder(enriched, typeMapper, builderClass)
+                .addCode(verifyNotBuilt())
```

Comment:
Code gen vs JVM inlining this? Feels a bit bloaty

Reply:
Ya I got lazy -- should use a private verifier method.